Account/FxAClient10.swift

```diff
@@ -48,6 +60,37 @@ public struct FxASignResponse {
     }
 }
 
+public struct FxAStatusResponse {
+    let exists: Bool
+    let locked: Bool
+
+    init(exists: Bool, locked: Bool) {
+        self.exists = exists
+        self.locked = locked
+    }
+}
+
+public struct FxADevicesResponse {
+    struct FxADevice {
```

Comment:
Heh, another artifact from 4 contributors working on the same bug! I created a separate `FxADevice` class (in `FxADevice.swift`) that has these fields already, so we should probably just use that here and drop this.